src/BlazorWebFormsComponents/ImageButton.razor

```diff
@@ -1,3 +1,58 @@
 @inherits ButtonBaseComponent
+@{
+	var hasToolTip = !string.IsNullOrEmpty(ToolTip);
+}
 
-<input type="image" src="@ImageUrl" alt="@AlternateText" longdesc="@DescriptionUrl" @onclick="Click" />
+@if (Visible)
```

Comment:
Hint: This needs to be improved, if there's away to create an element dynamicall yand bind events to it